content/en/about/technical-stack.md

```diff
@@ -5,38 +5,45 @@ position: 8
 category: 'About Us'
 ---
 
-Continuously investing in quality (functional programming, code quality, tests, pull request reviews, refactoring…) is part of our strategy to
-sustainably maximize the business value we deliver.
+Continuously investing in quality (functional programming, code quality, tests, pull request reviews, refactoring, etc.)
+is part of our strategy to sustainably maximize the business value we deliver.
 
 Here's where we are now:
 
 ## Frontend
 
-- Our webapp frontend projects are written in `Vuejs`, `Sass` and `Pug`. We make use of modern `JavaScript`
-- We maintain a UI Library that is used in all of our frontend projects.
-- We rely heavily on functional `JavaScript` patterns. 
+- Our webapp frontend projects are written in Vue.js, Sass, and Pug, using modern JavaScript.
+- We maintain [a UI Library](https://github.com/narrative-io/tackle-box) that is used in all of our frontend projects.
 
 ## Backend
 
-We sit somewhere in the middle of the “Scala as a worse Haskell” and “Scala as a better Java” spectrum.  We love
-functional programming and we do make use of libraries like `cats`, `cats-effect`, `http4s`, `doobie`, but keep a pragmatic 
-stance on purity and are also exposed to the `AWS Java SDK`/`hadoop`/`spark` ecosystems. 
+We sit somewhere in the middle of the “Scala as a worse Haskell” and “Scala as a better Java” spectrum. We love
+functional programming and make use of libraries like `cats`, `cats-effect`, `http4s`, `doobie`, but we keep a
+pragmatic stance on purity and have to play nice with the Hadoop and Spark ecosystems.
 
 Here are some highlights of the major backend components:
-- `open-api`: an `http4s` API running on `Fargate` to submit (buy, sell) orders to the marketplace
-- `ingestion`: spark jobs that ingest supplier data and store it in the data lake and puddles (Apache Iceberg).
-- `transaction`: spark jobs that transact the data between the buyers and sellers
-- `delivery`: spark jobs triggered by `AWS Step Functions` that deliver the transacted data to a format suitable for the buyers' consumption
+
+- `api`: an `http4s` API running on Fargate that is used by any person or process interacting with our platform (e.g.
```

Comment:
Tried to update this a bit but don't feel like I did a great job (e.g. `apps` is too wordy). If you can think of some better, more communicative bullet points lmk.